packaging/installer/functions.sh

```diff
@@ -234,7 +231,7 @@ find_processors() {
     gnproc && return
   fi
 
-  local cpus
+  cpus
```

Comment:
Unless we’re doing something below with this variable that cares about the difference between an undefined variable and one that is defined but null, we should not need it declared here.